app/models/derived_variables/sales_log_variables.rb

```diff
@@ -102,6 +115,14 @@ def set_derived_fields!
         mortgage: 0,
       },
     },
+    {
+      conditions: {
+        mortgageused: 3,
+      },
+      derived_values: {
```

Comment:
Previously the mortgage value was being left as whatever it was before when switching to "don't know".

Reply:
Looks like it should.